src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -325,6 +325,175 @@ std::optional<Values> Visitor::visit(Parser::ValuesClauseContext* ctx) {
   return visitOptional(ctx->dataBlock());
 }
 
+// ____________________________________________________________________________________
+ParsedQuery Visitor::visit(Parser::UpdateContext* ctx) {
+  visit(ctx->prologue());
+
+  if (ctx->update()) {
+    reportNotSupported(ctx->update(),
+                       "Multiple updates in one query are not supported.");
+  }
+  return visit(ctx->update1());
```

Comment:
Add a test for this failure (it shouldn't be hard)

Reply:
Already covered by `expectUpdateFails("INSERT DATA { <a> <b> <c> } ; INSERT { ?a <b> <c> } WHERE { <d> <e> ?a }");`